### PR TITLE
Adds sentry to kaws

### PR DESCRIPTION
**Pull request by xtina-starr**
Addresses: [GROW-1120](https://artsyproduct.atlassian.net/browse/GROW-1120)

This adds error tracking for the kaws project.

ToDo:
- [x]  Setup Kaws projects in Sentry for staging and production.
- [ ]  Set the environment variables.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -10,12 +10,14 @@ setupDataDog()
...
 import "reflect-metadata"
 
+import * as Sentry from "@sentry/node"
 import { GraphQLServer, Options } from "graphql-yoga"
 import { parse } from "mongodb-uri"
 import * as morgan from "morgan"
 import { Connection, createConnection } from "typeorm"
 import { databaseConfig } from "./config/database"
 import { createSchema } from "./utils/createSchema"
+const enableSentry = !!SENTRY_PRIVATE_DSN
```
Comment (eessex) [inline src/index.ts]: what is this `!!` syntax?

Comment (l2succes) [reply src/index.ts]: it's the same as `Boolean(SENTRY_PRIVATE_DSN)` it basically converts an truthy or falsy value to a proper boolean

File: src/index.ts [hunk 2]
```diff
@@ -48,6 +50,19 @@ async function bootstrap() {
       debug: NODE_ENV === "development",
     }
 
+    // Setup Sentry
+    if (enableSentry) {
+      Sentry.init({
+        dsn: "https://caca4ef0651e4adbbd1f099702dbb779@sentry.io/1405985",
```
Comment (l2succes) [inline src/index.ts]: @xtina-starr we should use the env variable for this correct?

Comment (xtina-starr) [reply src/index.ts]: oh shoot I actually set it up that way and for got to remove this

File: src/index.ts (hunk 1 above)
Comment (xtina-starr) [reply src/index.ts]: `Boolean(SENTRY_PRIVATE_DSN)` might actually be better.

Comment (eessex) [reply src/index.ts]: Agreed that seems more communicative.

File: src/index.ts (hunk 2 above)
Comment (xtina-starr) [reply src/index.ts]: Good catch @l2succes! This was for the staging environment. I'll go ahead and generate a new key.

**Comment by xtina-starr** [comment]
@eessex I don't think so especially considering there is a conditional check for the presence of the env variable. I just added the ToDo to remind myself.

**Pull request merged**